dependencies/jessie/kafo_parsers/control

```diff
@@ -11,6 +11,7 @@ Package: ruby-kafo-parsers
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
 Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter,
-         puppet (<< 4.0.0)
+         ruby-json,
```

Comment:
This shouldn't be needed with Ruby 1.9 or higher as JSON's built into Ruby, so all apart from precise can probably have it removed.